### PR TITLE
Implement nova get_version

### PR DESCRIPTION
**Pull request by Benjamin91300**
Recent novaclient versions and openstackclient require resource
/compute/v2. This commit provides it.

### CODE REVIEW

**Comment by BenjamenMeyer** [comment]
Along with #618 this should resolve #511

File: mimic/test/test_nova.py [hunk 1]
```diff
@@ -184,6 +193,23 @@ def setUp(self):
             create_server(self.helper, name=self.server_name))
         self.server_id = self.create_server_response_body['server']['id']
 
+    def test_get_version_nova(self):
+        """
+        Check response when listing available version of nova.
```
Comment (glyph) [inline mimic/test/test_nova.py]: Could you add a citation for this that points at Nova's docs?

**Comment by glyph** [comment]
Please just address the minor test doc issue and land.

File: mimic/test/test_nova.py (hunk 1 above)
Comment (BenjamenMeyer) [inline mimic/test/test_nova.py]: http://developer.openstack.org/api-ref-compute-v2.1.html

**Comment by Benjamin91300** [comment]
@glyph, @BenjamenMeyer:
I added a reference to the nova doc in docstring.

I hope it is what you expected.

**Comment by BenjamenMeyer** [comment]
@Benjamin91300 I would add it to the handler of the response itself, not simply the test (https://github.com/rackerlabs/mimic/pull/619/files#diff-623976324cdaaa34626dd686efc298b9R34).

Once that and the current merge conflict are resolved then I'll merge it as it otherwise LGTM.

**Comment by BenjamenMeyer** [comment]
:+1:

**Pull request merged**